src/components/crown/crownMultiselect/crownMultiselect.vue

```diff
@@ -70,9 +77,13 @@ export default {
     highlightedShapes: () => store.getters.highlightedShapes,
   },
   methods: {
-    copyElement() {
-      // @todo: Implement copyElement
-      this.$emit('copy-nodes');
+    copySelection() {
+      // @todo: Implement copySelection
+      this.$emit('copy-selection');
+    },
+    duplicateSelection() {
+      // @todo: Implement duplicateSelection
```

Comment:
you can remove the todos. Same for line 81

Reply:
Removed